ios/brave-ios/Sources/Brave/WebFilters/CustomFilterListStorage.swift

```diff
@@ -172,7 +172,7 @@ import WebKit
     guard
       let folderURL = FileManager.default.getOrCreateFolder(
         name: "custom_rules",
-        location: .applicationDirectory
+        location: .applicationSupportDirectory
```

Comment:
What is the purpose of this change ? Is it the part making it faster ? Using the support directory instead of main directory

Reply:
Basically because using the application directory was a mistake. You can't save files there and the screen has errors and loading the file has errors. This is really the main problem in this PR.